### ISSUE TITLE
Plugin: ConditionalPaging

### ISSUE THREAD

**Issue by skorunka** [DataTables/Plugins#286]
Why jQuery methods Show/Hide are not used for showing/hiding a pager?

Currently it is using "$paging.css('visibility', 'hidden');" which does not hide the "white-space" area reserved by paginate controls.

"$paging.css('display', 'none');" and "$paging.css('display', 'inherit');" would also sold the problem.

**Comment by DataTables** [comment]
I think that is exactly why it is using `visibility` - so it doesn't cause the page to jump around as the controls change. Feel free to modify to suit your needs.

**Comment by skorunka** [comment]
Yeah, I know what you mean. but it looks weird if you have a white-space on you page. I did change it, but how about having it configurable?

**Comment by DataTables** [comment]
Sounds like a good idea - pull requests welcome :-)
